addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1032,16 +1224,71 @@ MockServer.include({
             };
         });
     },
+    /**
+     * Simulates `_message_compute_author` on `mail.thread`.
+     *
+     * @private
+     * @param {string} model
+     * @param {integer[]} ids
+     * @param {Object} [context={}]
+     * @return {integer}
```

Comment:
Wrong, should be array

Reply:
good catch, should be `returns` as well, not sure from which I copied it but I got it wrong everywhere, I'll change them all.